apps/transaction/templates/transaction/partials/_room_category_list.html

```
@@ -1,115 +1,104 @@
+                          hx-indicator="#body-loading-spinner"
+                          hx-swap="outerHTML"
+                          class="d-flex flex-column gap-3">
+                      {% csrf_token %}
+                      <input type="hidden" name="action" value="update" />
+                      <input type="hidden" name="room_category_id" value="{{ room_category.id }}" />
+                      <div class="input-group input-group-sm align-items-center">
+                        <span class="input-group-text" id="order-label-{{ room_category.id }}">{% trans "Order" %}</span>
+                        {% with failed_id=failed_room_category_id|stringformat:"s" %}
+                          {% if room_category_update_form and failed_id == room_category.id|stringformat:"s" %}
+                            {% with update_for_category=room_category_update_form %}
+                              <input type="number"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)